shopify_python/google_styleguide.py

```diff
@@ -228,3 +237,15 @@ def __use_simple_list_comp(self, node):  # type: (astroid.ListComp) -> None
         """List comprehensions are okay to use for simple cases."""
         if len(node.generators) > 1:
             self.add_message('complex-list-comp', node=node)
+
+    def __use_cond_expr(self, node):  # type: (astroid.If) -> None
+        """Only one liner conditional expressions"""
+
+        if (len(node.body) == 1) and (len(node.orelse) == 1):
+            try:
+                if_body_name = node.body[0].targets[0].name
+                else_body_name = node.orelse[0].targets[0].name
+            except AttributeError:
```

Comment:
We don't generally follow that advice, but it has a point. I'd definitely like to see something more precise than `except AttributeError`, because I don't know if it's because `node` doesn't have a `body`, `node` doesn't have a `orelse`, etc etc etc.

You can check if `isinstance(node.body[0].targets[0], AssignName)` is true instead, if you think that's cleaner. Then you're guaranteed to have a `.name`.